Makefile

```diff
@@ -53,6 +54,7 @@ __check_defined = \
 # Linker flags
 VERSION_VARIABLES := -X $(REPOPATH)/pkg/crc/version.crcVersion=$(CRC_VERSION) \
 	-X $(REPOPATH)/pkg/crc/version.bundleVersion=$(BUNDLE_VERSION) \
```

Comment:
`bundleVersion` could be renamed to `openshiftVersion` for more clarity in a follow-up commit.

Reply:
@cfergeau yes that something I can do a follow up commit